photo.go

```diff
@@ -65,6 +94,70 @@ func (p *ImgixPhoto) GetThumbnailForWidthAndHeight(w, h int) string {
 	return fullUrl.String()
 }
 
+func (p *ThumborRaw) GetPhotoForWidth(w int) string {
+	thumborOptions := gothumbor.ThumborOptions{Width: w, Smart: true}
+	thumborPath, err := gothumbor.GetCryptedThumborPath(p.Secret, p.Key, thumborOptions)
+	if err != nil {
+		fmt.Print(err)
```

Comment:
Maybe use `log.Error` and then return "". Otherwise the function will try to continue with an incorrect/empty `thumborPath`.